scripts/generate_bottle.sh

```diff
@@ -1,15 +1,15 @@
 #! /bin/bash
 
-VERS=`sw_vers -productVersion | awk '/10\.14\..*/{print $0}'`
+VERS=`sw_vers -productVersion | awk '/10\.15\..*/{print $0}'`
 if [[ -z "$VERS" ]]; then
-   VERS=`sw_vers -productVersion | awk '/10\.15.*/{print $0}'`
+   VERS=`sw_vers -productVersion | awk '/11\.*.*/{print $0}'`
```

Comment:
Could you test this part ?       awk in line 5 looks a * more than required, not sure it will work you need test it.

Reply:
My bad, it's a typo. Just fixed.